### PR TITLE
Bump bundler to 2.2.33

### PR DESCRIPTION
**Pull request by codihuston**
### Desired Outcome

Bump bundler to 2.2.33 for security reasons.

### Implemented Changes

- _What's changed? Why were these changes made?_

   Updated dockerfiles for base images:

  - ubi-ruby-fips
  - ubuntu-ruby-fips
  - phusion-ruby-fips

- _How should the reviewer approach this PR, especially if manual tests are required?_
- _Are there relevant screenshots you can add to the PR description?_

### Connected Issue/Story

Resolves #[relevant GitHub issue(s), e.g. 76]

CyberArk internal issue ID: [CONJSE-1738]

### Definition of Done
*At least 1 todo must be completed in the sections below for the PR to be
merged.*

#### Changelog

- [x] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a
  CHANGELOG update

#### Test coverage

- [ ] This PR includes new unit and integration tests to go with the code
  changes, or
- [x] The changes in this PR do not require tests

#### Documentation

- [ ] Docs (e.g. `README`s) were updated in this PR
- [ ] A follow-up issue to update official docs has been filed here: [insert issue ID]
- [x] This PR does not require updating any documentation

#### Behavior

- [ ] This PR changes product behavior and has been reviewed by a PO, or
- [ ] These changes are part of a larger initiative that will be reviewed later, or
- [x] No behavior was changed with this PR

#### Security

- [x] Security architect has reviewed the changes in this PR,
- [ ] These changes are part of a larger initiative with a separate security review, or
- [ ] There are no security aspects to these changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [2.0.7] - 2023-05-16
+
+### Security
+- Update bundler to 2.2.33
+  [cyberark/conjur-base-image#111](https://github.com/cyberark/conjur-base-image/pull/111) to remove CVE-2021-43809
```
[the remark this answers is not in the document]
Comment (codihuston) [reply CHANGELOG.md:11]: It all renders as one line, I was just following the format of previous entries.

[the remark this answers is not in the document]
Comment (micahlee) [reply CHANGELOG.md:11]: Right, but I think it should be 

```md
- Update bundler to 2.2.33 to remove CVE-2021-43809.
  [cyberark/conjur-base-image#111](https://github.com/cyberark/conjur-base-image/pull/111)
```

[the remark this answers is not in the document]
Comment (codihuston) [reply CHANGELOG.md:11]: Ohhh, I understand. I'll fix that. Thanks!

**Review by micahlee — APPROVED** [review]
LGTM!

**Pull request merged**